build/dockerfiles/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN /dashboard/airgap.sh -i /dashboard/packages/devfile-registry/air-gap/index.j
 
 FROM docker.io/node:18.19.1-alpine3.19
 
-RUN apk --no-cache add curl
+RUN apk --no-cache add curl git
```

Comment:
Don’t forget to include Git in [the downstream image](https://gitlab.cee.redhat.com/codeready-workspaces/devspaces-images/-/blob/main/devspaces-dashboard/build/dockerfiles/rhel.Dockerfile?ref_type=heads)